providers/cpp/cpp.go

```diff
@@ -0,0 +1,21 @@
+package bing
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("cpp", &CppProvider{})
```

Comment:
Curious if this should be cplusplus, to match the site name. It will allow "cp" to be used as shorthand based on the expansion logic.